Enable builds on posix

melund opened the pull request.

melund committed:
Enable builds on posix

melund committed:
Use SHLIB_EXT to make posix/mac compatible

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [28]

melund committed:
Fix lint error

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

melund commented:
@conda-forge-admin, please re-render

melund commented:
@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you but ran into some issues, please ping conda-forge/core for further assistance. You can also try [re-rendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you but ran into some issues, please ping conda-forge/core for further assistance. You can also try [re-rendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).

melund committed:
Update build number

melund committed:
MNT: Re-rendered with conda-build 3.18.11, conda-smithy 3.6.5, and conda-forge-pinning 2020.01.16

melund committed:
Specify numpy include dir and version manually

melund committed:
drop using patches and point directly to a fork of BTK.

This solves the problem with newline problem
where patches either worked on win or unix, but not both

melund committed:
fix prefix and c++ 14

melund committed:
Add git as test dependency

melund committed:
Fix file path case in test case

melund committed:
copy lib files as well

melund committed:
no quotes

melund committed:
Try other way of coping files to site packages
